packages/okta-angular/src/okta/okta.service.ts

```diff
@@ -97,8 +97,13 @@ export class OktaAuthService {
 
     /**
      * Launches the login redirect.
+     * @param fromUri
+     * @param additionalParams
      */
-    loginRedirect(additionalParams?: object) {
+    loginRedirect(fromUri: String, additionalParams?: object) {
```

Comment:
This signature change is also a breaking change

Reply:
Should this be pulled out as another PR?